Documentation/policy/language.rst

```diff
@@ -1077,9 +1077,12 @@ each selected node, they apply only to the host namespace, including
 host-networking pods. They therefore don't apply to communications between
 non-host-networking pods and locations outside of the cluster.
 
-The following policy will allow ingress traffic for any node with the label
-``type=worker`` on TCP ports 22, 6443 (kube-apiserver), 2379 (etcd) and 4240
-(health checks), as well as UDP port 8472 (VXLAN).
+The following policy example would allow ingress traffic for any node with the
+label ``type=ingress-worker`` for remote access (TCP 22), kube-apiserver
```

Comment:
Nit: If we are keeping the text for the SSH port only, maybe this should state:
```suggestion
label ``type=ingress-worker`` for remote access (e.g., TCP 22), kube-apiserver
```
because users may actually use a different port.